src/components/speakers/SpeakerCard.js

```diff
@@ -21,16 +21,16 @@ export default ({
       <p className="speaker__title">{`${title}, ${company}`}</p>
       <p>{introText}</p>
       <div>
-        <a
+        {githubUrl !== '#' && <a
           className="github__icon"
           href={githubUrl}
           target="_blank"
-        />
-        <a
+        />}
+        {twitterUrl !== '#' && <a
```

Comment:
What if we make the urls an empty string if it has no link? That way, we can simplify this check to `{ url && component }`

Reply:
Sure, was rushing to catch a bus 😂